config/modules/featureExtractor.yaml

```diff
@@ -0,0 +1,13 @@
+# @package _global_
+FeatureExtractor:
+  _target_: gtsfm_cfgnode.FeatureExtractor
```

Comment:
Thanks for the PR, @xwu4lab, this is cool.

Could we have a single hierarchal yaml file instead of 4 different ones? i think would be more readable

Reply:
According to your comment, I also provide an example called gtsfm_cfgnode_from_single_file.py to make it work.